vocab/credentials/v2/template.html

```diff
@@ -0,0 +1,241 @@
+    <section id="abstract">
+      <p>This document describes the
+        <span property="dc:title"><%= ont["dc:title"]["en"] %></span>
+        and Term definitions used for describing Verifiable Credentials [[VC-DATA-MODEL]]. 
+        This document provides the RDFS [[RDF-SCHEMA]] vocabulary definition.
+         <!-- and a description of the JSON-LD context definition for use with defining Verifiable Credentials. -->
+      </p>
+      <p>Alternate versions of the vocabulary definition exist in
+        <a rel="alternate" href="credentials.ttl">Turtle</a> and
+        <a rel="alternate" href="credentials.jsonld">JSON-LD</a>.
+        <!-- which also includes the <code>@context</code> required for metadata descriptions. -->
+        <!--These versions may also be retrieved from <code>FIXME</code> using an appropriate HTTP <em>Accept</em> header.-->
+      </p>
+      <dl>
+        <dt>Published:</dt><dd><time property="dc:date"><%=ont["dc:date"]%></time></dd>
+        <%- unless Array(ont["owl:imports"]).empty?%>
+        <dt>Imports:</dt>
+        <%- Array(ont["owl:imports"]).each do |ref| %>
+          <dd><a href="<%=ref%>" property="owl:imports"><%=ref%></a></dd>
+        <% end -%>
+        <% end -%>
+        <dt>Version Info:</dt>
+        <!-- <dd><a href="<%=ont['owl:versionInfo']%>" property="owl:versionInfo"><%=ont['owl:versionInfo']%></a></dd> -->
+        <dd>2.0</dd>
+        <dt>See Also:</dt>
+        <% Array(ont["rdfs:seeAlso"]).each do |ref| %>
+          <dd><a href="<%=ref%>" property="rdfs:seeAlso"><%=ref%></a></dd>
+        <% end %>
+      </dl>
+    </section>
+    <section id="sotd">
+      <p>
+        Comments regarding this document are welcome. Please file issues
+        directly on <a href="https://github.com/w3c/vc-data-model/issues/">GitHub</a>, or send them to
+        <a href="mailto:public-vc-comments@w3.org">public-vc-comments@w3.org</a>
+        (<a href="mailto:public-vc-comments-request@w3.org?subject=subscribe">subscribe</a>,
+        <a href="https://lists.w3.org/Archives/Public/public-vc-comments/">archives</a>).
+      </p>
+      <!-- <p>
+        The Working Group seeks implementation feedback, having set the
+        requirement of at least two implementations of each feature as the
+        exit criteria for the Verifiable Credentials Data Model 2.0 
+        Candidate Recommendation phase.  The group aims
+        to obtain reports from two producers and two consumers for each
+        feature if possible.  For details, see
+        the <a href="https://github.com/w3c/vc-test-suite/">test suite</a> and
+        sample <a href="https://w3c.github.io/vc-test-suite/implementations/">implementation
+        report</a>.
+      </p> -->
+    </section>
+    <section>
+      <h2>Introduction</h2>
+      <p property="dc:description"><%= ont["dc:description"]["en"] %></p>
+      <p>This specification makes use of the following namespaces:</p>
```

Comment:
I really hope we can simplify this list in v2.... especially removing odrl.

Reply:
Note that this PR has not yet looked at the v2 version of the `@context`. In fact, it is only an attempt to clean up the 1.1 vocabulary.

I agree that the ODRL reference may be an overkill.